docs/reference/ml/ml-shared.asciidoc

```diff
@@ -507,6 +507,357 @@ tag::data-frame-analytics-stats[]
 An array of statistics objects for {dfanalytics-jobs}, which are
 sorted by the `id` value in ascending order.
 
+//Begin analysis_stats
+`analysis_stats`::
+(object)
+An object containing statistical data about the analysis.
++
+.Properties of `analysis_stats`
+[%collapsible%open]
+====
+//Begin classification_stats
+`classification_stats`:::
+(object)
+An object containing statistical data about the {classanalysis}.
++
+.Properties of `classification_stats`
+[%collapsible%open]
+=====
+//Begin class_hyperparameters
+`hyperparameters`::::
+(object)
+An object containing the hyperparameters of the {classanalysis}.
++
+.Properties of `hyperparameters`
+[%collapsible%open]
+======
+`class_assignment_objective`::::
+(string)
+Defines whether class assignment maximizes the accuracy or the minimum recall 
+metric. Possible values are `accuracy` and `minimum_recall`.
+
+`downsample_factor`::::
+(double)
+The value of the downsample factor.
+
+`eta`::::
+(double)
+The value of the eta hyperparameter.
```

Comment:
Since `eta` can be specified explicitly in the request body, maybe you can cross-reference it here?